integration/testrunner/grpc.go

```diff
@@ -119,19 +125,28 @@ func testGRPCEndpoint(ctx context.Context, t *testing.T, g *GRPC) {
 	t.Run("hello_req_unary_success", func(t *testing.T) {
 		t.Parallel()
 		id := uuid.New().String()
+		t.Logf("using uuid %s", id)
 		_, err := g.TalkerClient.Hello(ctx, &talkerpb.HelloRequest{Message: "Some Message", Target: id})
 		if err != nil {
 			t.Errorf("could not greet: %v", err)
 		}
-		query := g.makeQueryForGRPCUnary(id)
-		queryIfAuditLogExistsWithRetries(ctx, t, query, g.Config, "unary_hello")
+		query := g.makeQueryForGRPC(id)
+		t.Log(query.Q)
```

Comment:
Do we need such log? Is it just for debug?

Reply:
I think it could be useful to debug when tests fails. So I am in favor of keeping this log.

Reply:
Can you use t.Logf and add some context info to indicate what the print log is?

Reply:
Done

Reply:
Can you use t.Logf instead of call t.Log twice?
like `t.Log("querying bigquery table using the following statement: %v", query.Q)`